### ISSUE TITLE
having a file with 'public' in the name get filename changed

### ISSUE THREAD

**Issue by ThatTomPerson** [JeffreyWay/laravel-mix#342]
I have a file named `publications.js` and all that gets output is `ations.js`

**Comment by JeffreyWay** [comment]
Is this in a stand-alone app? Can you give me a quick set of steps to reproduce this?

**Comment by ThatTomPerson** [comment]
just was testing out mix with our work project so sadly I can not share what I had

but the webpack.mix.js file looked like
```
mix.js('resources/assets/js/app.js', 'dist/')
   .js('resources/assets/js/publications.js', 'dist/')
   .sass('resources/assets/sass/app.scss', 'dist/')
   .sourceMaps()
   .version();
```

**Comment by JeffreyWay** [comment]
Thanks so much. This was fixed, as part of 5f4a0061bb37538950f47752050238e69bd45e42.
